### ISSUE TITLE
GR downgraded?

### ISSUE THREAD

**Issue by tkf** [JuliaPlots/Plots.jl#2027]
The latest REQUIRE had `GR 0.39.0` https://github.com/JuliaPlots/Plots.jl/commit/9774eca8e27556d89062cc47c8a0c1a6b028b441 while Project.toml now has

https://github.com/JuliaPlots/Plots.jl/blob/f742ae4c8850b6d9d665df84f06226882c33be47/Project.toml#L34

I don't see any discussion in #2022.  Is it an accident or intentional?

**Comment by daschw** [comment]
To be honest, it was not intentional from my side. Is this causing any issues?

**Comment by tkf** [comment]
I noticed it only because I pin GR to 0.36 due to https://github.com/jheinen/GR.jl/issues/163 and now my sysimage test fails with [```julia: contourf.c:313: marching_squares: Assertion `xi == i && yi == j' failed.```](https://travis-ci.com/tkf/julia-sysimage-recipes/jobs/199449894#L665) after Plots.jl is upgraded to the latest version.

But I just realized that it is impossible to fix this at Plots.jl side (and it's easy to workaround this at user side).  Pkg would pick up Plots.jl v0.25.0 even if v0.25.1 were released with a fixed GR bound.  I'm closing this because there is no actionable fix in Plots.jl side.  Sorry about the noise.
